kolibri/core/apps.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriCoreConfig(AppConfig):
+    name = 'kolibri.core'
+
+    def ready(self):
+        from django.db import connection
+
+        # Only run the PRAGMAs if we're using SQLite
+        if connection.vendor == "sqlite":
+            cursor = connection.cursor()
+            cursor.execute("PRAGMA synchronous=OFF;")
```

Comment:
@rtibbles whoops, just added a new commit while you were writing this. I still kept synchronous mode to OFF, to check how much speed gains we get from the last pragma benchmark we ran.

Just waiting for another buildkite build now, then benchmarking it again.